.travis.yml

```diff
@@ -2,6 +2,7 @@ language: ruby
 rvm:
   - 2.6.6
```

Comment:
I know that Ruby 2.6.8 and 2.7.4 have been released the other day (while I was away from this project...).
I'll send another PR for updating them.